src/urbit/lib/chess.hoon

```diff
@@ -1,4 +1,4 @@
-/-  chess
+/-  chess, historical
```

Comment:
It seems like `sur` files have to be imported into `lib`. When I add the `/-` in `app/chess` it doesn't compile.

Reply:
It absolutely compiles, I just tested it. "It doesn't compile" is unclear and vague: what exactly is the error?

If it's a syntax error about the `/-` rune, then make sure that the import is before the `/+` rune: the `fas` runes have a strict ordering (the order in which they appear in the [documentation](https://developers.urbit.org/reference/hoon/rune/fas)).

If it's a `-find` error, then make sure you're either:
1. Referencing the correct namespace for the type (e.g. `state-0.historical`)
2. Implicitly referencing the correct namespace by stripping it during import (e.g. `/-  *historical`)
3. Implicitly referencing the correct namespace by explicitly stripping it (e.g. `=,  historical`)

Reply:
Thanks, I didn't know about the ordering for `fas` runes so I was getting a syntax error from the `/-`when I added it after the `/+`.

For the sake of my understanding, importing `sur/historical` into `lib` seemed to work and make those types accessible in `app`, but I assume it's better for style and clarity to import the file directly where it is being referenced?

Reply:
Yeah, I think that would be best - the only thing that complicates it is that I'm not sure if the compiler has any sort of import deduplication. For example, if we imported `historical` into both `lib/chess` and `app/chess`, would we have two copies of everything in our subject? This is important, because in another language, we would import `sur/chess` into both `lib/chess` and `app/chess`, trusting that duplicates would be filtered. Instead, we have to have the more complicated import rule of "import files wherever they're directly used, unless another file we're importing has already imported them."